config/kernel-monitor.json

```diff
@@ -26,7 +26,7 @@
 		{
 			"type": "temporary",
 			"reason": "TaskHung",
-			"pattern": "task \\S+:\\w+ blocked for more than \\w+ seconds\\."
+			"pattern": "task [\\S\\s]+:\\w+ blocked for more than \\w+ seconds\\."
```

Comment:
This basically means matching everything.

I guess we only want to match \S + space.

Reply:
but it was unable to identify the cases where there is a space like this task airflow scheduler:

Reply:
i mean it is technically \S + space + \S + ":*"

Reply:
I mean, we can do something like `[\\S ]+`?

Reply:
yes we can use this